ui-tests/src/test/resources/features/integrations/integration-import-export.feature

```diff
@@ -5,7 +5,7 @@ Feature: Integration - Import Export
 
   Background: Clean application state
     Given clean application state
-    And reset content of "contact" table
+    Given reset content of "contact" table
```

Comment:
"And" is correct step here - And has no meaning by itself - it inherits previous type of step. In this case the step is "Given". We did not respect this rule in the early developing phase but now we try to be more consistent with it so feel free to correct every scenario step you are working on. You can find more info about steps here: https://docs.cucumber.io/gherkin/reference/

Reply:
Updated